README.md

```diff
@@ -1,26 +1,13 @@
-# Terraform [Dead Man's Snitch](https://deadmanssnitch.com/) Provider
-
-Now available via the [Terraform Registry](https://registry.terraform.io/providers/plukevdh/dmsnitch/latest)!
-
-## Note on Compatability
-
-This branch is meant to work against the latest version of Terraform. For previous versions, see below
-
-- 0.0.x: Terraform 0.11 and earlier
-- 0.1.x: Terraform 0.12+ 
-
 ## Requirements
 
-- Terraform 0.12.x or higher
+- Terraform 0.13.x or higher
 - Go 1.11 or higher
 - Go module support
 
-Please note [the following details](https://www.terraform.io/docs/extend/terraform-0.12-compatibility.html) if you have built this plugin prior to Terraform 0.12. 
-
 ## Setting up the provider
 
 ```sh
-$ go get -u github.com/plukevdh/terraform-provider-dmsnitch
```

Comment:
These changes will not be accepted if you want them merged upstream for the registry.